server/src/Irony.cpp

```diff
@@ -245,7 +245,8 @@ void Irony::complete(const std::string &file,
     std::cout << "(\n";
 
     // re-use the same buffers to avoid unnecessary allocations
-    std::string typedtext, brief, resultType, prototype, postCompCar;
+    std::string typedtext, brief, resultType, prototype, postCompCar, access;
```

Comment:
availability is a var name already used. i have used available - but this does exceed the 80 char limit ... ill let you decide